.env.docker

```
@@ -59,7 +59,7 @@ DC_MINIO_SECRET_KEY=medusaappsecret_change_me
 # Postgres container + bootstrap role setup
 DC_POSTGRES_SUPERUSER=root
 DC_POSTGRES_SUPERUSER_PASSWORD=root
-DC_POSTGRES_PUBLIC_PORT=5432
+DC_POSTGRES_PUBLIC_PORT=5433
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DC_POSTGRES_PUBLIC_PORT key should go before the DC_POSTGRES_SUPERUSER key

(UnorderedKey)